src/pages/publication/references_tab/components/InstitutionSearch.tsx

```diff
@@ -15,19 +15,21 @@ interface InstitutionSearchProps {
   dataTestId: string;
```

Comment:
fyi: tror denne fila kommer til å bli overflødig snart. Institutions returnerer alle institutions (uten søkestreng) så kan vi filtrere de med Autocomplete i frontent, og gjøre nytt oppslag for subunits når man velger institution

Reply:
ok 👌